README.md

```diff
@@ -1,2 +1,26 @@
 # toolhunt
-A web application for editing Toolhub records in a fun and easy way
+# Description
+A web application for editing Toolhub records in a fun and easy way.
+It allows users to view and add missing fields for tools in Toolhub.
+This is an Outreachy Internship project.
+
+## Issue Tracker Link 
+ Click [View Site](https://phabricator.wikimedia.org/project/board/6283/)  to visit the site
+
+
+
+
+## Setup/Installation Requirements
+* Clone this repo to your machine
+* `git clone https://github.com/wikimedia/toolhunt.git` 
+* `cd into toolhunt`
+
+
+## Technologies to be Used
+* Vue.js
+* Flask
+* HTML
+* CSS
+* Bootstrap
```

Comment:
Bootstrap: As we haven't discussed component libraries yet, I'd leave this out for now.